lib/rubycritic/generators/json/simple.rb

```diff
@@ -24,6 +25,18 @@ def data
             score: @analysed_modules.score
           }
         end
+
+        def file_directory
+          @file_directory ||= Pathname.new(Config.root)
+        end
+
+        def file_name
+          'report.json'
+        end
+
+        def file_pathname
+          File.join(file_directory, file_name)
```

Comment:
Merged #file_name and #file_pathname to #file_pathname. I used File.join because it looks simpler to me, can substitute it with  Pathname.new(file_directory, FILE_NAME), should I?